internal/engine/wire.go

```diff
@@ -23,22 +40,11 @@ func provideBuildAndDeployer(
 	shouldFallBackToImgBuild func(error) bool) (BuildAndDeployer, error) {
 	wire.Build(
 		// dockerImageBuilder ( = ImageBuilder)
-		build.DefaultImageBuilder,
-		build.NewDockerImageBuilder,
 		build.DefaultConsole,
 		build.DefaultOut,
 		wire.Value(build.Labels{}),
 
-		// ImageBuildAndDeployer (FallbackBuildAndDeployer)
-		wire.Bind(new(FallbackBuildAndDeployer), new(ImageBuildAndDeployer)),
-		NewImageBuildAndDeployer,
-
-		// FirstLineBuildAndDeployer (LocalContainerBaD OR SyncletBaD)
-		build.NewContainerUpdater, // in case it's a LocalContainerBuildAndDeployer
-		NewFirstLineBuildAndDeployer,
-
-		wire.Bind(new(BuildAndDeployer), new(CompositeBuildAndDeployer)),
-		NewCompositeBuildAndDeployer,
+		DeployerWireSet,
```

Comment:
```
build.DefaultConsole,
build.DefaultOut,
wire.Value(build.Labels{}),
```
^ believe all this stuff is used exclusively by imageBuildAndDeployer, any reason why it's not also in the DeployerWireSet?